puppet/modules/jenkins_job_builder/files/centos.org/jobs/foreman-luna-test.yml

```diff
@@ -1,41 +1,50 @@
 ---
 - job-template:
-    name: foreman-{foreman_version}-{distro}-release-test
+    name: foreman-luna-{version}-{distro}-{action}-test
     project-type: 'pipeline'
     sandbox: true
     parameters:
       - string:
-          name: expected_version
-          default: 'UNKNOWN'
-      - string:
-          name: foreman_version
-          default: '{foreman_version}'
+          name: version
+          default: '{version}'
       - string:
           name: distro
           default: '{distro}'
+      - string:
+          name: action
+          default: '{action}'
+      - string:
+          name: type
+          default: 'katello'
```

Comment:
Isn't luna its own type?

Reply:
It is. Just wanted to comment the same. Also the luna groovy files can be deleted now